packages/http-client-python/emitter/src/code-model.ts

```diff
@@ -323,5 +323,8 @@ export function emitCodeModel(sdkContext: PythonSdkContext) {
     ...simpleTypesMap.values(),
   ];
   codeModel["crossLanguagePackageId"] = ignoreDiagnostics(getCrossLanguagePackageId(sdkContext));
+  if ((sdkContext.emitContext.options as any).flavor === "azure") {
```

Comment:
Any test cases that we could verify this change?

Reply:
_metadata.json is not part of package so we can't test it by import then check. You can see the generated file in PR https://github.com/Azure/autorest.python/pull/3089/files#diff-3cd63a63f5c9e91001bdea81bb22b04fb20951ffae50393999131961cd61af6b

Reply:
i see.  So for python, we only include the value of api-version?